src/test/kotlin/ApplicationLocal.kt

```diff
@@ -20,7 +20,6 @@ fun main(args: Array<String>) {
                       "mock-infotrygd-replika",
                       "mock-kodeverk",
                       "mock-blankett",
-                      "mock-iverksett",
-                      "mock-brev")
```

Comment:
Burde default være att mock-brev er med? Sånn att man ikke må starte opp brev og familie-dokument for att få igennom ett flyt?

Reply:
Ja. Var ikke meningen at det skulle ut 😅 

Men lurer egentlig på om vi burde ha en ApplicationLocalBrev, eller noe sånt, for må endre den hver gang.